Tests/NIOTests/ChannelTests.swift

```diff
@@ -237,8 +243,8 @@ public class ChannelTests: XCTestCase {
             }
             /* assert that the canary values are still okay, we should definitely have never written those */
             XCTAssertEqual(managed.last!.toOpaque(), Unmanaged.passUnretained(o).toOpaque())
-            XCTAssertEqual(0xdeadbeef, Int(bitPattern: iovecs.last!.iov_base))
-            XCTAssertEqual(0xdeadbeef, iovecs.last!.iov_len)
+            XCTAssertEqual(0x1337beef, Int(bitPattern: iovecs.last!.iov_base))
+            XCTAssertEqual(0x1337beef, iovecs.last!.iov_len)
```

Comment:
Out of interest why does the old value prevent compilation?

Reply:
Int(0xdeadbeef) (which the compiler generates for literals) overflows `Int32` (which then asserts). The other option which I think I actually used in the original patch is `0xdeadbeef as UInt` or sth like that. I don't mind either.

Reply:
Might be better to use that other version, just for consistency with ourselves (we use `deadbeef` as our marker value elsewhere in the code)

Reply:
I'd propose `0xdedbeef` or `0xdadbeef` or smth